### PR TITLE
fix(devicegrid): continuous calls to mps

### PR DESCRIPTION
**Pull request by Mahendra-intel**

### CODE REVIEW

File: src/reactjs/components/DeviceGrid/DeviceGrid.tsx [hunk 1]
```diff
@@ -85,7 +89,9 @@ export class DeviceGrid extends React.Component<gridProps, gridStates>{
 
         updateConnectionSocket.onclose = () => {
             this.retry_timer = setInterval(() => {
-                this.connectionStatusControl();
+                // this.connectionStatusControl();
```
Comment (rsdmike) [inline src/reactjs/components/DeviceGrid/DeviceGrid.tsx:92]: remove commented code

File: src/reactjs/components/DeviceGrid/DeviceGrid.tsx [hunk 2]
```diff
@@ -76,6 +79,7 @@ export class DeviceGrid extends React.Component<gridProps, gridStates>{
     connectionStatusControl = () => {
         let protocol = window.location.protocol === "https:" ? "https" : "http";
         const updateConnectionSocket = new WebSocket(window.location.protocol.replace(protocol, "wss") + "//" + this.props.mpsServer + "/notifications/control.ashx")
```
Comment (rsdmike) [inline src/reactjs/components/DeviceGrid/DeviceGrid.tsx:81]: why does it matter what protocol the site is served on? Especially if its going to be replaced with "wss". Update this to be: 
```
const updateConnectionSocket = new WebSocket(`wss://${this.props.mpsServer}/notifications/control.ashx`)
```

**Review by rsdmike — CHANGES_REQUESTED** [review]

**Review by madhavilosetty-intel — APPROVED** [review]

File: src/reactjs/components/DeviceGrid/DeviceGrid.tsx (hunk 1 above)
Comment (Mahendra-intel) [reply src/reactjs/components/DeviceGrid/DeviceGrid.tsx]: done

File: src/reactjs/components/DeviceGrid/DeviceGrid.tsx (hunk 2 above)
Comment (Mahendra-intel) [reply src/reactjs/components/DeviceGrid/DeviceGrid.tsx]: done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**